### ISSUE TITLE
Member role is unable to unmerge issues when "Let Members Delete Events" is disabled

### ISSUE THREAD

**Issue by narsaynorath** [getsentry/sentry#52906]
### Environment

SaaS (https://sentry.io/)

### Steps to Reproduce

1. Disable "Let Members Delete Events"
2. Merge two issues together with an account with a Member role
3. Attempt to unmerge the issue with a Member role

### Expected Result

The issue is successfully queued for unmerging.

### Actual Result

The request to unmerge the issue returns a 403 status and the issue cannot be unmerged.

### Product Area

Issues

**Comment by lobsterkatie** [comment]
This does indeed seem like a bug, which I've added to the grouping backlog.

Note: There's [a comment from 8.5 years ago](https://github.com/getsentry/sentry/blob/f039bc95a19fb9eae867c5138d78634734e87299/src/sentry/api/helpers/group_index/update.py#L647) from Cramer, suggesting that merging should maybe be its own endpoint. If we followed that suggestion (and then moved unmerging to the same endpoint, or a related one), this problem would go away, because it'd turn into a `PUT` request rather than a `DELETE` request.
